subliminal/providers/subscenter.py

```diff
@@ -191,6 +192,7 @@ def query(self, title, season=None, episode=None):
                         subtitle_key = subtitle_item['key']
                         downloaded = subtitle_item['downloaded']
                         release = subtitle_item['subtitle_version']
+                        h_version = subtitle_item['h_version']
```

Comment:
@Diaoul It looks fine, the `test_download_subtitle` fails in python3.x.
It looks like the test depends on ordering within the Subtitle object and somewhere it is not the same between Python 2.x to Python 3.x (I've used python 3.5 to test and it passed with the same value as in Python 2.x)
Any ideas ?